README.md

```diff
@@ -9,15 +9,17 @@
 * [Technologies Used](#Technologies-Used)
 
 ## Description
-Welcome to What's Cookin! - your go-to website for cooking and baking inspiration. Discover a wide range of dinner ideas and delightful desserts. On our home page, you'll find the Recipe of the Day and a featured category. Feel free to use the search bar, browse by category, or explore all recipes. Each recipe provides a list of ingredients, total price, and directions. Save your favorite recipes by clicking the ❤️ icon in the top right corner and visit your saved recipes on the "Saved Recipes" page.
+Welcome to What's Cookin! - your go-to website for cooking and baking inspiration. Discover a wide range of dinner ideas and delightful desserts. On our home page, you'll find the Recipe of the Day and a featured category. Feel free to use the search bar, browse by category, or explore all recipes. Each recipe provides a list of ingredients, total price, and directions. Save your recipes to cook by clicking the ❤️ icon in the top right corner and visiting your saved recipes on the "Saved Recipes" page. Once saved, a grocery list is displayed on the "Saved Recipes" page showing quantities of the required ingredients to cool all your saved recipes along with a total budget. 
```

Comment:
For the last sentence, small typo of "cook" instead of "cool". Maybe we can say "... page showing a grocery list of all the ingredients to cook all your saved recipes along with a total budget."

Reply:
Good catch!